qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -549,6 +579,263 @@ def __deepcopy__(self, _memo: dict = None) -> "IBMBackend":
+                which are the terminal instruction to a qubit. ESP readout can offer higher fidelity
+                than standard measurement sequences. See
+                `here <https://arxiv.org/pdf/2008.08571.pdf>`_.
+                Default: ``True`` if backend supports ESP readout, else ``False``. Backend support
+                for ESP readout is determined by the flag ``measure_esp_enabled`` in
+                ``backend.configuration()``.
+            noise_model: Noise model. (Simulators only)
+            seed_simulator: Random seed to control sampling. (Simulators only)
+            **run_config: Extra arguments used to configure the run.
+
+        Returns:
+            The job to be executed.
+
+        Raises:
+            IBMBackendApiError: If an unexpected error occurred while submitting
+                the job.
+            IBMBackendApiProtocolError: If an unexpected value received from
+                 the server.
+            IBMBackendValueError:
+                - If an input parameter value is not valid.
+                - If ESP readout is used and the backend does not support this.
+        """
+        # pylint: disable=arguments-differ
+        validate_job_tags(job_tags)
+        if not isinstance(circuits, List):
+            circuits = [circuits]
+        self._check_circuits_attributes(circuits)
+
+        if use_measure_esp and getattr(self.configuration(), "measure_esp_enabled", False) is False:
+            raise IBMBackendValueError(
+                "ESP readout not supported on this device. Please make sure the flag "
+                "'use_measure_esp' is unset or set to 'False'."
+            )
+        actually_dynamic = are_circuits_dynamic(circuits)
+        if dynamic is False and actually_dynamic:
+            warnings.warn(
+                "Parameter 'dynamic' is False, but the circuit contains dynamic constructs."
+            )
+        dynamic = dynamic or actually_dynamic
+
+        if dynamic and "qasm3" not in getattr(self.configuration(), "supported_features", []):
+            warnings.warn(f"The backend {self.name} does not support dynamic circuits.")
+
+        status = self.status()
+        if status.operational is True and status.status_msg != "active":
+            warnings.warn(f"The backend {self.name} is currently paused.")
+
+        program_id = str(run_config.get("program_id", ""))
+        if program_id:
+            run_config.pop("program_id", None)
+        else:
+            program_id = QASM3RUNNERPROGRAMID if dynamic else QOBJRUNNERPROGRAMID
+
+        image: Optional[str] = run_config.get("image", None)  # type: ignore
+        if image is not None:
+            image = str(image)
+
+        if isinstance(init_circuit, bool):
+            warnings.warn(
+                "init_circuit does not accept boolean values. "
+                "A quantum circuit should be passed in instead."
+            )
```

Comment:
(can be separate PR) - this should be an error and not a warning.

Reply:
Changed to an error. I didn't think it was worth adding a test.